core/src/main/java/com/google/errorprone/bugpatterns/UnusedVariable.java

```diff
@@ -149,6 +149,8 @@ public final class UnusedVariable extends BugChecker implements CompilationUnitT
   // TODO(ghm): Find a sensible place to dedupe this with UnnecessarilyVisible.
```

Comment:
If acceptable I don't mind also updating [this list](https://github.com/google/error-prone/blob/b9e01a8d4f2a7a822eaf6b8f95f72f2f0ca72912/core/src/main/java/com/google/errorprone/bugpatterns/UnnecessarilyVisible.java#L59-L64). Alternatively we can upstream our [`RefasterRuleModifiers`](https://github.com/PicnicSupermarket/error-prone-support/blob/87f79dae60bd1be34f71ebeeba7d783670f96e70/error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/RefasterRuleModifiers.java) check, which normalizes modifier usage by Refaster rule definitions more generally.

Reply:
> If acceptable I don't mind also updating [this list](https://github.com/google/error-prone/blob/b9e01a8d4f2a7a822eaf6b8f95f72f2f0ca72912/core/src/main/java/com/google/errorprone/bugpatterns/UnnecessarilyVisible.java#L59-L64).

Thanks, updating the list until we get around to deduping them SGTM

> Alternatively we can upstream our [`RefasterRuleModifiers`](https://github.com/PicnicSupermarket/error-prone-support/blob/87f79dae60bd1be34f71ebeeba7d783670f96e70/error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/RefasterRuleModifiers.java) check, which normalizes modifier usage by Refaster rule definitions more generally.

That also sounds good if you have time/interest, but it also doesn't need to block this PR

Reply:
> Thanks, updating the list until we get around to deduping them SGTM

Top; will do.

> That also sounds good if you have time/interest, but it also doesn't need to block this PR

I'll make a note to consider this later then :heavy_check_mark:. (When we contributed #3813 and #2232 that caused some issues for downstream users due to a name collision (PicnicSupermarket/error-prone-support#622 and PicnicSupermarket/error-prone-support#686), so maybe this time I'll try to find an alternative name; hehe.)